### PR TITLE
remove Python 2 and Python 3.5 support

### PR DESCRIPTION
**Pull request by haroldrandom**
Remove Python 2 and Python 3.5 in CI and Docs

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -40,14 +40,13 @@
         'Environment :: Console',
         'License :: OSI Approved :: MIT License',
         'Natural Language :: English',
-        'Programming Language :: Python :: 2.7',
         'Programming Language :: Python :: 3.5',
```
Comment (Juliehzl) [inline setup.py]: we should abandon python 3.5, right?

File: tox.ini [hunk 2]
```diff
@@ -1,6 +1,5 @@
 [tox]
 envlist =
-    py27
     py35
```
Comment (Juliehzl) [inline tox.ini]: abandon py35

Comment (haroldrandom) [reply tox.ini]: fixed

File: setup.py (hunk 1 above)
Comment (haroldrandom) [reply setup.py]: fixed

**Comment by ctaggart** [comment]
What prompted the removal of Python 3.5? I ask because my Azure Cloud Shell is still has python 3.5.2.

**Pull request merged**